policyengine_us/variables/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefit/hi_cdcc_eligible_income_floor.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class hi_cdcc_eligible_income_floor(Variable):
```

Comment:
Let's skip the floor amount here since it's a tax-unit-level variable, we can do that in the `hi_cdcc_min_head_spouse_earned` formula instead. Make this a bool instead.
```suggestion
class hi_cdcc_income_floor_eligible(Variable):
```